homework/test/test-list.js

```diff
@@ -1,8 +1,14 @@
 describe('Test > ', function() {
+  var subject;
+
   beforeEach(function() {
+    subject = new TestManager;
   });
 
-  it('will test some pure functions', function() {
+  it('will test some pure functions', function(done) {
     // Write any pure function assertion here.
+    subject.set(3, 5);
+    assert.deepEqual(subject.add(), 3+5);
```

Comment:
Good. Although you should make sure the file is included correctly when you executed the test.